gulpfile.js

```diff
@@ -1,29 +1,101 @@
+const { spawn } = require('child_process');
+
+const del = require('del');
 const forever = require('forever-monitor');
 const gulp = require('gulp');
+const gulpBabel = require('gulp-babel');
 const eslint = require('gulp-eslint');
 const mocha = require('gulp-mocha');
+const sourcemaps = require('gulp-sourcemaps');
+const webpackStream = require('webpack-stream');
```

Comment:
Could you add a blank line after this require to separate node_modules imports from local imports?